src/app/components/crypto-assets/crypto-currency-asset/crypto-currency-asset-item.layout.tsx

```diff
@@ -31,44 +33,50 @@ export function CryptoCurrencyAssetItemLayout({
   onClick,
   rightElement,
   usdBalance,
+  isLoading,
 }: CryptoCurrencyAssetItemLayoutProps) {
   const { balance, dataTestId, formattedBalance, title } =
     parseCryptoCurrencyAssetBalance(assetBalance);
 
+  const titleRight = isLoading ? (
```

Comment:
Can you refactor this so it's not a nested ternary statement? 

You can even just move the `isLoading` check lower to separate it a bit:
```
const titleRight =  rightElement ? (
    rightElement
  ) : (
    <BasicTooltip
      asChild
      label={formattedBalance.isAbbreviated ? balance.amount.toString() : undefined}
      side="left"
    >
...

  const content = (
    <ItemLayout
      flagImg={icon}
      titleLeft={title}
      captionLeft={balance.symbol}
      titleRight={isLoading ? <BalanceShimmer width="126px" /> : titleRight}
      captionRight={captionRight}
```

Reply:
Also wonder if there's a cleaner way to do this? ~Isn't there a way to wrap balance simmer over the whole thing?~ Realise this is the skeleton

Reply:
What about a wrapper component with API like this

```ts
<SkeletonLoader isLoading={isLoading}>
  <ItemLayout {...props} />
</SkeletonLoader>
```

Reply:
yeah, made this as wrapper component